### PR TITLE
Implement Footer component and Logo

### LINKED ISSUE #1: Static Landing Page
# Story 
As a user, I want to visit the homepage so that I can see a a navigation bar, a mission statement, categories, and a footer with an about us. 

As a sighted visitor, I will see a navigation bar, the mission statement, and three categories to choose from. At the bottom of the page, there is a footer. When I try to click on something, nothing happens. 

### Concerns 
- [x] Are all sizes on elements relative to screen size (Mobile first)
- [x] What's our color scheme? 
- [x] When do we implement testing? 

Please see issues: #2 #3 #4 #5  for a breakdown of each element of App.

### LINKED ISSUE #4: Footer & About Us Components
# Story 
As a user, I want to visit the home page so that I can see a footer with links to creators linkedIn & githubs. 

As a sighted user, I see at the bottom of the page there's a footer that contains the clickable links to the creators. This *is* clickable. I see three creators and there are links to their gh pages & their LinkedIn pages. 

### Concerns 
- [ ] Mobile first viewing
- [ ] What information do we want displayed here? 
- [x] ~~a separate component for each of us? Or hardcoded?~~

### PR DESCRIPTION
**Pull request by conconartist**
- [x] Feature
- [x] Refactor
- [x] Bug Fix 
- [x] Clean Up

Logo icon works and shows up correctly! Footer component is added and updated.  Fair Trade category mentions are changed to "Allergy Friendly".

Updated Logo component for the SVG to refactor for color and size correctly.  Footer component was updated with style attributes and links to each contributor's social media pages.  App component was refactored to add the Logo, Footer, and category name change from "Fair Trade" to "Allergen Friendly" (this includes link name changes, labels, and photo).

In the Components folder, App was updated in state and in the rendering.  The folders for Footer and Logo were added and updated with JS and CSS files.

Open the project directory and enter `npm start` in terminal to view the changes in the browser.  Scroll to the bottom to see the footer and click on links.

If we want to, we can keep updating the mission statement and sizing of the icon.  Otherwise, I think we should be ready to move on! 

#### Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code follows the style guidelines of this project
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const Footer = () => {
+    return (
+       <footer>
+           <h4>FaceIt</h4> 
+             <h5>is brought to you by</h5>
+              <div className="dev-container">
+                <div className="dev">
+                  <p>Kelsie Besinger Yeh</p>
+                  <div className="links">
+                    <a href="https://github.com/kelsiebesingeryeh/">Github</a>
```
Comment (gaj23) [inline src/components/Footer/Footer.js:14]: What do you think about having icons/svgs instead of just plain links? I like how a lot of websites use that.

**Review by gaj23 — COMMENTED** [review]
This wraps up our iteration 1 right on target! Thank you.

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (kelsiebesingeryeh) [reply src/components/Footer/Footer.js:14]: agreed, it is pretty easy to do! I did this for my personal website

File: src/components/App/App.js [hunk 2]
```diff
@@ -64,7 +65,7 @@ class App extends Component {
             <div className="title-container">
               <h1>FaceIt</h1>
               <Logo />
-              <h2 class="mission-statement">Discover beauty products that compliment our lifestyles.</h2>
+              <h2 class="mission-statement">Discover beauty products that compliment your lifestyle.</h2>
```
Comment (kelsiebesingeryeh) [inline src/components/App/App.js:68]: I think this should be className instead of class. I had an error in my console about this one

**Review by kelsiebesingeryeh — APPROVED** [review]

File: src/components/App/App.js (hunk 2 above)
Comment (conconartist) [reply src/components/App/App.js:68]: Oh, you're right! Thank you!

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (conconartist) [reply src/components/Footer/Footer.js:14]: Yes! I was actually intending to do that originally but went down a rabbit hole looking at icons, so I just put the links up as a placeholder for now and forgot about it.  Thanks for the reminder!

**Pull request merged**